Quadriga/src/main/resources/velocitytemplates/transformationdetails.vm

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<graphml xmlns="http://graphml.graphdrawing.org/xmlns"  
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"

```

Comment:
for some reason during transformation, the xmlns and xmlns:xsi declarations are getting lost, so that the graphml that is produced is invalid:
```
<?xml version="1.0" encoding="UTF-8" ?>
<graphml xsi:schemaLocation="http://graphml.graphdrawing.org/xmlns http://graphml.graphdrawing.org/xmlns/1.0/graphml.xsd">
<graph id="http://www.digitalhps.org/concepts/CON6bb6bd78-9030-42da-8ba0-f93bfbeb933f" edgedefault="undirected">
<node id="APP_EVN-4589959468852165194">
...
```